src/Result.h

```diff
@@ -64,4 +66,33 @@ using TResult = std::variant<GOOD_TYPE, Error>;
  */
 using TResultOpt = std::optional<Error>;
 
+/** @brief Checks if given parameter contains alternative error value.
+ *         If so, it prints the containing error message.
+ * @return true if parameter contains error type, false otherwise
+ */
+template <class GOOD_TYPE>
+static bool checkAlternativeError(TResult<GOOD_TYPE> ret) {
+  if (std::holds_alternative<Error>(ret)) {
```

Comment:
is this working now?

Reply:
nope.. just removed it with https://github.com/skaupper/VirtualJukebox/pull/19/commits/8acd55e3d4bcbbce7d98675693a6bb6e73092bc6 ... i have absolutely no idea why this won't work....